src/main/resources/avro/metadata.avdl

```diff
@@ -114,4 +135,42 @@ record Dataset {
 
 }
 
+/**
+An analysis contains an interpretation of one or several experiments.
+(e.g. SNVs, copy number variations, methylation status) together with
+information about the methodology used.
+*/
+record Analysis {
+  /** The analysis UUID. This is globally unique. */
```

Comment:
@calbach - we took this from the Analysis record in master when we started to create the Variant Annotation API. It's currently unchanged, but it makes sense to change the comment to the one in metadata_intro.rst.
@mbaudis - we can't move to the new structure in the metadata branch as there often won't be information available for the required fields - for example individualIds, biosampleIds. At this stage, it would be simplest to use the old record the switch to something similarly light weight later when the Analysis model is updated. There is already the plan to look at Program and move this out of reads, so this may be a good way to go.